src/controllers/userController.ts

```diff
@@ -0,0 +1,60 @@
+import { Request, Response } from 'express';
+import User from '../models/userModel';
+
+export const getAllUsers = async (req: Request, res: Response): Promise<void> => {
+  const { page = 1, limit = 10 } = req.query;
+
+  try {
+    const users = await User.find()
+      .limit(Number(limit) * 1)
+      .skip((Number(page) - 1) * Number(limit))
+      .exec();
```

Comment:
.exec(); is not necessary

Reply:
Typescript issues warning regarding unnecessary await without .exec(). Didn't find another way to workaround this.